mitodl/open-discussions
# Always show current user at top of mods list

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes #1132 

#### What's this PR do?
Places the current user at the top of the moderators list 

#### How should this be manually tested?
Add several users to a moderators list.  Log in as each of those users and go to the moderators list page. "You are a moderator of this channel" should always be at the top of the list.
